cmd/spiffe-aws-assume-role/cli/credentials_cmd.go

```diff
@@ -27,11 +28,13 @@ type CredentialsCmd struct {
 	SessionDuration time.Duration `optional:"" type:"iso8601duration" help:"AWS session duration in ISO8601 duration format (e.g. PT5M for five minutes)"`
 	LogFilePath     string        `optional:"" help:"Path to log file"`
 	TelemetrySocket string        `optional:"" help:"Socket address (TCP/UNIX) to emit metrics to (e.g. 127.0.0.1:8200)"`
+	SentryDSN       string        `optional:"" help:"DSN from Sentry for sending errors"`
 	Debug           bool          `optional:"" help:"Enable debug logging"`
```

Comment:
An example DSN (but not a real one) might be helpful just so people who have no context (like myself) have an idea of what they might be looking for.